libavformat/imfdec.c

```diff
@@ -370,14 +372,12 @@ static int open_track_resource_context(AVFormatContext *s,
         return ret;
     }
 
-    ret = avformat_find_stream_info(track_resource->ctx, NULL);
```

Comment:
@valnoel I think this is no longer necessary if the files referenced by the ASSETMAP are required to be MXF files, right?